src/content/docs/apm/agents/c-sdk/instrumentation/distributed-tracing-c-sdk.mdx

```diff
@@ -0,0 +1,273 @@
+---
+title: 'Distributed tracing for your C services'
+tags:
+  - Agents
+  - C SDK
+  - Instrumentation
+metaDescription: 'For the New Relic C SDK: how to implement distributed tracing across your C language applications and services.'
+redirects:
+  - /docs/agents/c-sdk/instrumentation/enable-distributed-tracing-your-c-applications
+  - /docs/apm/agents/c-sdk/instrumentation/enable-distributed-tracing-your-c-applications
+---
+
```

Comment:
Won't this will need the EOL banner like other c-sdk related pages?  ex: https://docs.newrelic.com/docs/apm/agents/c-sdk/instrumentation/enable-distributed-tracing-your-c-applications/
```
EOL NOTICE
As of April 2022, we're discontinuing support for several capabilities, including the C SDK. For more details, including how you can easily prepare for this transition, see our [Explorers Hub post](https://discuss.newrelic.com/t/q1-bulk-eol-announcement-fy23/181744).
```